web_frontend/cloudscheduler/glintwebui/glint_utils.py

```diff
@@ -195,21 +203,90 @@ def get_image(config, image_name, image_checksum, group_name, cloud_name=None):
         if len(image_candidates) > 0:
             return image_candidates[0]
         else:
-            #No image that fits specs
+            # No image that fits specs
             return False
     else:
-        #getting a specific image
+        # getting a specific image
         logging.debug("Retrieving image %s" % image_name)
-        where_clause = "group_name='%s' and cloud_name='%s' and name='%s' and checksum='%s'" % (group_name, cloud_name, image_name, image_checksum)
+        where_clause = "group_name='%s' and cloud_name='%s' and name='%s' and checksum='%s'" % (
+            group_name, cloud_name, image_name, image_checksum)
         rc, msg, image_candidates = config.db_query(IMAGES, where=where_clause)
         if len(image_candidates) > 0:
             return image_candidates[0]
         else:
-            #No image that fits specs
+            # No image that fits specs
             return False
 
 
 # at a length of 16 with a 85 symbol alphabet we have a N/16^85 chance of a collision, pretty darn unlikely
 def generate_tx_id(length=16):
     return ''.join(random.choice(ALPHABET) for i in range(length))
 
+
+def convert_sparsify_compress(src_file_path, virt_sparsify, with_compression):
+    if os.path.exists(src_file_path):
+        print("process starts")
+        added_cmd = ''
+        is_sparsified = ''
+        is_compressed = ''
+        # report_message = ''
+
+        if virt_sparsify:
+            print("Virt sparsify checkbox is selected")
```

Comment:
More prints in this function, they should be removed or turned into logging.debug statements.